contact.html

```diff
@@ -74,15 +75,14 @@
                 </div>
             </div>
         </section>
-        <footer>
+        <div class= "footer footer-fixed-bottom">
```

Comment:
This should stay a footer tag with `footer-fixed-bottom`as the class attribute. Why is the fixed bottom necessary? Is it a bootstrap css class or one you made? Because it doesn't actually do anything according to the browser inspector.

This should be something like:
`<footer class="footer-fixed-bottom">`